django_sorcery/db/url.py

```diff
@@ -15,6 +15,8 @@
     "sqlserver_ado": "mssql",
 }
 
+ENGINE_OPTIONS_NORMALIZATION = {"echo": lambda x: x in ["True", True]}
```

Comment:
allows for `ALIAS_URL=sqlite:///db.sqlite?engine_options_echo=True`, etc.

Reply:
I was planning on doing this at some point with `engine_` and `pool_` prefixes. `engine_options_` is a bit too verbose, we can drop down to just `engine_`, afaik no dialect has any querystring option with `engine_..`